File: javascript/Day4/GroceryManagerApp.js
const prompt = require("prompt-sync")(); 
let itemList = [];
hunterly = true;
while(hunterly){
console.log(`WELCOME TO YOUR GROCERY STORE
		   
		  
		  1.Add item
		  2.Remove item
		  3.Show Available item
		  0.exit`);
let option = parseInt(prompt("Enter your choice: "));
switch(option){
	case 1: 
		let userInput = prompt("Enter item to add: ");
		addMyItem(userInput);
		break;
	case 2: 
		userInput2 = prompt("Enter item to remove: ");
		removeMyItem(userInput2);
		break;
	case 3: 
		console.log(" Available item:");
		showItem();
		break;
	case 0:
	hunterly = false;
	console.log("bye bye");
	break;
	default:
            console.log("Invalid option. Please try again.");	


}
}



function addMyItem(addItem){
		itemList.push(addItem);
		console.log("item added:",itemList);
}
function removeMyItem(removeItem){
		let index = itemList.indexOf(removeItem);
		if(index !== -1){
		itemList.splice(index, 1);
		console.log("item removed:", itemList);
		} else {
		console.log("item not found");
	}
}

function showItem(){
itemList.forEach((item, index) => {
	console.log(`${index + 1}. ${item}`);

});
}

File: python/Day4/grocery_manager_app.py
new_array = [];
def add_my_item(add_item):
	new_array.append(add_item);
	return f"'{add_item}' has been added to your grocery list.";

def remove_my_item(remove_item):
	new_array.remove(remove_item);
	return f"'{remove_item}' has been removed from your grocery list.";

def to_show_item():
	if not new_array:
		return "your grocery List is empty";
	else:
		for item in new_array:
			return new_array;


def main():
	while(True):
		name = """ 
				WELCOME TO YOUR GROCERY STORE
		   
		  
		  		1.Add item
		  		2.Remove item
		  		3.Show Available item
		  		0.exit
				""";
		print(name);
		option = int(input("Enter a option:"))
		match(option):
			case 1:
				userInput = input("1.Add item:"); 
				print(add_my_item(userInput));
			

			case 2:
				userInput = input("2.Remove item:"); 
				print(remove_my_item(userInput));
			

			case 3:
				print(to_show_item());
			
			case 0:
				while False:
					print("bye bye");
				break;
			case _:
				print(" Invalid option. Please try");
			 
			
if __name__ == "__main__":
    main()
